src/components/forms/controls/Input/Input.module.scss

```diff
@@ -17,6 +17,7 @@
     display: flex;
     align-items: center;
     column-gap: bk.$spacing-1;
+    padding-block-start: 0.2rem; // Expand the target area to min 24px for accessibility compliance
```

Comment:
This causes misalignment within `MultiSearch`, where the text prefix is no longer baseline-aligned with the input.